### PR TITLE
Sc 18719/status

### PR DESCRIPTION
**Pull request by daniellemaxwell**
### Scope of changes

Corrects bug to display the correct `Status` in the member list table.

Fixes SC-18719

### Type of change

- [x] bug fix
- [ ] new feature
- [ ] documentation
- [ ] other (describe)

### Acceptance criteria

Describe how reviewers can test this change to be sure that it works correctly. Add a checklist if possible

### Author checklist

- [ ] I have manually tested the change and/or added automation in the form of unit tests or integration tests
- [ ]  I have updated the dependencies list
- [ ]  I have recompiled and included new protocol buffers to reflect changes I made
- [ ]  I have added new test fixtures as needed to support added tests
- [x]   Check this box if a reviewer can merge this pull request after approval (leave it unchecked if you want to do it yourself)
- [x]  I have moved the associated Shortcut story to "Ready for Review"

### Reviewer(s) checklist

- [ ] Any new user-facing content that has been added for this PR has been QA'ed to ensure correct grammar, spelling, and understandability.

### CODE REVIEW

File: web/gds-user-ui/src/modules/dashboard/member/__mocks__/index.ts
Comment (daniellemaxwell) [inline web/gds-user-ui/src/modules/dashboard/member/__mocks__/index.ts:14]: The updates were made to test `getVaspStatus`.

File: web/gds-user-ui/src/modules/dashboard/member/utils.ts
Comment (daniellemaxwell) [inline web/gds-user-ui/src/modules/dashboard/member/utils.ts:53]: The statuses come from here: https://github.com/trisacrypto/trisa/blob/main/pkg/trisa/gds/models/v1beta1/models.pb.go#L82C1-L82C1

Comment (masskoder) [reply web/gds-user-ui/src/modules/dashboard/member/utils.ts:53]: Would you mind updating L43-48 with this
```js
  const m = member.map((item: any) => {
    return {
      ...item,
      status : getVaspStatus(item.status)
      registered_directory: getVapsNetwork(item.registered_directory)
    };
  });

```

File: web/gds-user-ui/src/modules/dashboard/member/utils.ts
Comment (masskoder) [inline web/gds-user-ui/src/modules/dashboard/member/utils.ts:73]: it would be better to surround the returned value with the `t` macro for the translation string

**Review by masskoder — APPROVED** [review]
Looks good. I added some suggestions that need to be done.  i will approve but i will appreciate if you can make those changes before merging it.

File: web/gds-user-ui/src/modules/dashboard/member/utils.ts
Comment (daniellemaxwell) [reply web/gds-user-ui/src/modules/dashboard/member/utils.ts:53]: I've merged in the main branch so now this should be included.

**Pull request merged**
